### DISCUSSION TITLE
Issue Importing Layout from KLayout to Lumerical Interconnect

### DISCUSSION THREAD

**Discussion by schrodingertuller** [SiEPIC/SiEPIC-Tools#192] [Q&A]
I have a circuit in KLayout which contains Si waveguides, Si edge couplers, thermo-optic switches, electrical pads, and wireguides. When I go to the SiEPIC tab in the toolbar and navigate to 'Simulation, Circuits' > 'Circuit Simulation: Lumerical INTERCONNECT', Interconnect opens but in the output box I get an error saying that 'dcsource' is not found in the library. I have tried the same circuit without electrical pads and wireguides and still have the same issues. I have attached screenshots so you can see what I mean. Thanks a ton in advance to anyone who can provide insight.

Cheers
<img width="1728" alt="Screen Shot 2023-09-06 at 9 31 14 AM" src="https://github.com/SiEPIC/SiEPIC-Tools/assets/143816651/c28cff51-1d1e-4480-b21b-2120e830178b">
<img width="1728" alt="Screen Shot 2023-09-06 at 9 31 59 AM" src="https://github.com/SiEPIC/SiEPIC-Tools/assets/143816651/40912581-dcb9-4944-8db2-1a6925759bc8">

**Comment by mustafacc** [comment]
Hello @schrodingertuller
That's probably because the compact model library in the INTC library needed to simulate the AIM components you're using does not contain a dcsource element. My suggestion for a quick fix would be to add that element to the CML. it should already be there in the DRPSUNY AIM CML.

**Reply by schrodingertuller** [reply]
Thanks a ton, I'll try that
